script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt

```diff
@@ -851,62 +966,71 @@ The earth rises.
+After being bisected by the light,
+ one half the city begins to rise into the sky,
+ while the other sinks into the depths of the earth.
 }
 [sha:bf993f328315117f806fccce86be2c36af025fe1]{
 -- Page 332, Offset 39326.
 -- その断層のカタチは、開かれた獣の<顎|あご>のようだった。
-The shape of the fault is like the open jaws of a beast.
+The fault line resembles the open jaws of a beast.
 }
 [sha:1d7ae095d66cc79cf39dfee76a362e499ab327c2]{
 -- Page 333, Offset 39327.
 -- 「そ―――ま、さか。シエル、貴女」
-"Wait―――It can't be. Ciel, you..."
+"Wait... It can't be. Ciel, you―――"
 }
 [sha:6db9994817519df04557bc470b5b21901c87e619]{
 -- Page 333, Offset 39328.
 -- 「言ったでしょう。貴女は、貴女自身の重さで殺すと。
+// Uhh did you?
 "I told you, didn't I? That I would kill you using your own weight.
 }
 [sha:5eff62b0c102ee906d944e11199c4c7f0dd90d14]{
 -- Page 333, Offset 39329.
 -- 好きに暴れたかったのでしょうが、この街を自分と同じ<階|モ><梯|ノ>に変えた時は笑ってしまいました。馬鹿すぎて」
-You may want to run rampant as you wish, but seeing you change this city to be the same as yourself made me laugh. It was utterly foolish."
+I'm sure you just wanted to enjoy your little rampage,
+ but I couldn't help but laugh when I saw you transform this city into something the same Rank as yourself.
+ How utterly foolish."
 }
-[sha:1da3c45880d75c8cbe5d0291d4ee5a221870c07b]{
+[offset:39330]{
 -- Page 333, Offset 39330.
 -- 「―――！」
-"―――!"
+"%{i}Kh%{/i}―――!"
 }
 [sha:9702bf6f2ebf0cb4e26f0b88ba5a3fe0101d5647]{
 -- Page 334, Offset 39331.
 -- 「この質量ならもう逃げようがない。
-"With this mass, you can't run anymore.
+"With this much mass, you can't run anymore.
 }
 [sha:990dd984b49d52e71eb6262972c32ad17af3f1ff]{
 -- Page 334, Offset 39332.
 -- 貴女を満遍なく、その器ごと破壊してあげましょう。
-I'll completely destroy you along with your vessel.
+I'll thoroughly destroy you, along with your vessel.
 }
 [sha:61d8d5d07d5b89f4d81b0df20f82f7d3f6e4d444]{
 -- Page 334, Offset 39333.
 -- 再生に何年かかるか、それともこれで消滅するのか。これまでの自分の行いに<縋|すが>るのですね」
-Will it take years for you to recover? Or will this be your end? That will depend entirely on how you've acted until now."
+// Not sure what Ciel is referencing with the second part here yet, so how long a period should これまで imply? Fight? Longer?
+Will you be able to recover after a few years?
+ Or will this be the end of you?
+ It'll all depend on your actions up to this point."
 }
 [sha:5a4ff45a8e5623e851bac57c1836f0c9d7a69ab1]{
 -- Page 335, Offset 39334.
 -- 大きく開かれる……いや、上昇する星の<上顎|うわあご>。
-The upper jaws of the planet massively unfolds... no, it rises.
+The upper jaw of the planet opens wide――or rather, it ascends.
 }
 [sha:765cf73d9bffcfd620840a9dd9dcc9ec0ffaa93d]{
 -- Page 335, Offset 39335.
 -- <そ|・><れ|・>なら壊れる。再生や蘇生どころの話ではない。殺される前に、彼女を彼女たらしめるモノが壊される。
-With this, she will be destroyed. This isn't something she can regenerate or resurrect from. Before she is even killed, the thing that forms her very existence will be destroyed.
+%{i}This%{/i} will destroy her.
+ It isn't a matter of regeneration or resuscitation――the things that make her %{i}her%{/i} will be destroyed before she's killed.
```

Comment:
just call it resurrection. 

>%{i}This will%{/i} destroy her.
>It is beyond any regeneration or resurrection, That which makes her %{i}her%{/i} will be destroyed before she is killed.